Documentation/SwiftlyDocs.docc/swift-version.md

```diff
@@ -0,0 +1,97 @@
+
+swiftly is contributing to this effort by reading this file automatically. When switching toolchains it seeds them when you `swiftly install` or `swiftly use` a different toolchain.
+
+### Other versions: Swift Tools
+
+There are other sorts of versions in Swift. For a typical SwiftPM package, there's the "Swift Tools Version" at the top of the `Package.swift` file:
+
+```
+//swift-tools-version: 5.10
+
+Package(
+   name: "foo"
+)
+```
+
+From the [SwiftPM documentation](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#about-the-swift-tools-version) the first purpose of this version is to declare the version of the Package.swift file's contents so that it can be processed using the correct version of the package description structure. Another purpose is to set the minimum version of the "Swift tools" that are needed to use the package.
+
+From this first purpose, this has not so much to do with the toolchain version, as it does with the need to understand the package description that's inside the file. Newer toolchains will fall back to this version of the description to get the package description. It's also in the best interest of a package to use an older version for this so that downstream packages can continue to use older toolchains.
+
+In terms of the second purpose this acts as a lower-bound of the swift tools for reasons stated above. There are serious tensions between using this as the exact recommended version because there are additional concerns with developing, and testing this package that are not shared by consumers of it, such as features used in testing, formatting, and IDE integration.
+
+Not every project that uses a Swift toolchain uses SwiftPM. There are cases where CMake is being used as the build system. In other cases the toolchain is used to access a particular release of clang. Both swiftly and the `.swift-version` file can be used in these cases too.
+
+## Other versions: Swift Language Version
+
+There is also a version that you can set for the Swift language. For example, SwiftPM allows you to set them for a [package](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#swiftversion) in your `Package.swift` file.
+
+```
+Package(
+    name: "foo",
+    swiftLanguageVersions: [v5, v6]
+)
+```
+
+This version is used to set the supported language modes in the Swift compiler because it can support handling source code from much older, and newer versions of the language. This has little to do with the recommended toolchain version to use for a project, and is much more about supporting legacy Swift code. There are similar settings for the C, and C++ language standards used by clang.
+
+Newer toolchains can support much older language versions. For Swift versions, they tend to be only the major version, not the minor, and patch versions that uniquely identify an exact recommended toolchain.
```

Comment:
```suggestion
```

I think I'd drop this completely - the definition of what language mode is supported is in the docs linked above, and this reads as a repeat of the paragraph above.